src/AWS.Deploy.CLI/ServerMode/Models/SetConfigSettingsInput.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+
+namespace AWS.Deploy.CLI.ServerMode.Models
+{
+    public class SetConfigSettingsInput
+    {
+        public Dictionary<string, string> UpdatedSettings { get; set; } = new Dictionary<string, string>();
```

Comment:
I expect dictionary value to be `object`, how is it going to work with nested option settings.

If the key is path to the leaf setting, then it is worth calling `JsonPath` in docs to make it clear for the readers.

Can you also add some documentation how is it working?

Reply:
I added more documentation on the operation.